pages/linux/libreoffice.md

```diff
@@ -0,0 +1,20 @@
+# libreoffice
+
+> Use the powerful and free office suite, LibreOffice, from the command line.
```

Comment:
Perhaps this will read better?
```suggestion
> CLI for LibreOffice, a free office suite.
```

Reply:
The wording doesn't matter too much to me. I think Powerful a good descriptor for LibreOffice so I've reworded it with it included.

Reply:
Generally in tldr pages we try to avoid subjective terms such as `powerful`, or `easy-to-use` in the descriptions.